ietf-optical-impairment-topology.tree

```diff
@@ -28,15 +29,21 @@ module: ietf-optical-impairment-topology
                 |  +--ro amplifier
```

Comment:
the element container could be suppressed, since we have already amplifer container.
maybe for another round of change